dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -169,7 +169,10 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 								padSides={false}
 								shouldCenter={false}
 							>
-								<HeaderAdSlot />
+								<HeaderAdSlot
+									isPaidContent={!!front.config.isPaidContent}
+									shouldHideReaderRevenue={false}
```

Comment:
Flagging this here as it appears it's not possible to obtain a shouldHideReaderRevenue flag from a front.